web/src/components/SubscriptionPopup.jsx

```diff
@@ -207,29 +208,41 @@ export const SubscriptionPopup = (props) => {
             <ListItemIcon>
               <LockOpen fontSize="small" />
             </ListItemIcon>
-
             {t("action_bar_reservation_delete")}
           </MenuItem>
         )}
         <MenuItem onClick={handleSendTestMessage}>
           <ListItemIcon>
             <Send fontSize="small" />
           </ListItemIcon>
-
           {t("action_bar_send_test_notification")}
         </MenuItem>
         <MenuItem onClick={handleClearAll}>
           <ListItemIcon>
             <ClearAll fontSize="small" />
           </ListItemIcon>
-
           {t("action_bar_clear_notifications")}
         </MenuItem>
+        {!!subscription.mutedUntil && (
+          <MenuItem onClick={() => handleSetMutedUntil(0)}>
+            <ListItemIcon>
+              <Notifications fontSize="small" />
+            </ListItemIcon>
+            {t("action_bar_unmute_notifications")}
+          </MenuItem>
+        )}
+        {!subscription.mutedUntil && (
+          <MenuItem onClick={() => handleSetMutedUntil(1)}>
+            <ListItemIcon>
+              <NotificationsOff fontSize="small" />
+            </ListItemIcon>
+            {t("action_bar_mute_notifications")}
+          </MenuItem>
+        )}
```

Comment:
I would do `{subscription.mutedUntil ? ( ... ) : ( ... )}`. The `!!` is unnecessary since the ternary will coerce to Boolean.

Reply:
A ternary for long stuff like this is very hard to read. And readability is king king king 👑

Reply:
style preference I guess, but I find the way prettier formats the jsx ternaries pretty readable. 

anyway `&&` also coerces to boolean, so personally would just do `{subscription.mutedUntil && (...)}` then because double negation always makes me do a double take, but this is just nitpicking